server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/SetMessagesMethodTest.java

```diff
@@ -47,6 +47,7 @@
 import java.util.Date;
```

Comment:
You miss "copy" wich also increase the quota, and thus needs to be tested.

Reply:
we discussed that on mattermost yesterday, but we found that I doesn't increase quota. Now that I think about it, we mailbox sharing, it can. Is it the case you think about ?

Reply:
No, it's not.

The quota is incremented one time for each mailbox a mail is in. (That is each mailbox + uid). 

CF the test that I added